src/modules/ERC20TokenLimitModule.sol

```diff
@@ -27,29 +20,24 @@ import {BaseModule, IERC165} from "./BaseModule.sol";
 /// token contract
 contract ERC20TokenLimitModule is BaseModule, IExecutionHookModule {
     using UserOperationLib for PackedUserOperation;
-    using EnumerableSet for EnumerableSet.AddressSet;
-    using AssociatedLinkedListSetLib for AssociatedLinkedListSet;
 
     struct ERC20SpendLimit {
         address token;
-        uint256[] limits;
+        uint256 limit;
```

Comment:
Is this struct only used in installation?

Reply:
Yes that's right.